winml/test/common/googleTestMacros.h

```diff
@@ -72,11 +72,13 @@
   WINML_SKIP_TEST("GPU tests disabled because this is a WinML only build (no DML)")
 #define GPUTEST_ENABLED alwaysFalse()
 #else
-#define GPUTEST                                                                             \
-  if (auto no_gpu_tests = RuntimeParameters::Parameters.find("noGPUtests");                 \
-      no_gpu_tests != RuntimeParameters::Parameters.end() && no_gpu_tests->second != "0") { \
-    WINML_SKIP_TEST("GPU tests disabled");                                                  \
-  }
+#define GPUTEST                                                                               \
+  do {                                                                                        \
+    if (auto no_gpu_tests = RuntimeParameters::Parameters.find("noGPUtests");                 \
+        no_gpu_tests != RuntimeParameters::Parameters.end() && no_gpu_tests->second != "0") { \
+      WINML_SKIP_TEST("GPU tests disabled");                                                  \
+    }                                                                                         \
+  } while (0)
```

Comment:
Does this change also needs to be propagated to the SKIP_EDGECORE macro?

Reply:
I'll look through the macros and see which ones have slight inconsistencies